dialogs.c

```diff
@@ -1196,15 +1409,15 @@ void dialogs_init(GtkBuilder *builder)
 		if (ctx) {
 			iio_context_destroy(ctx);
 		} else {
-			GtkWidget *local = GTK_WIDGET(gtk_builder_get_object(
+/*			GtkWidget *local = GTK_WIDGET(gtk_builder_get_object(
```

Comment:
It this commented block is not needed anymore, it should be removed.

Reply:
fixed.